api/src/helpers.rs

```diff
@@ -0,0 +1,80 @@
+use basic_system::system_implementation::flat_storage_model::bytecode_padding_len;
+use basic_system::system_implementation::flat_storage_model::AccountProperties;
+use forward_system::run::PreimageSource;
+use ruint::aliases::U256;
+use std::alloc::Global;
+use zk_ee::execution_environment_type::ExecutionEnvironmentType;
+use zk_ee::utils::Bytes32;
+
+// Getters
+
+/// Retrieves balance from an account.
+pub fn get_balance(account: &AccountProperties) -> U256 {
+    account.balance
+}
+
+/// Retrieves nonce from an account.
+pub fn get_nonce(account: &AccountProperties) -> u64 {
+    account.nonce
+}
+
+/// Retrieves code for an account.
+/// This function returns unpadded code, without artifacts.
+pub fn get_code<P: PreimageSource>(
+    preimage_source: &mut P,
+    account: &AccountProperties,
```

Comment:
I think this function doesn't need to exist. However, there should be a function for splitting a preimage (`&[u8]`) into its parts.

Reply:
Is there where you need that but can't use this function?

Reply:
This one is quite useful, but it makes sense. I broke it down into two helpers.

Reply:
I didn't realize that AccountProperties are needed to be able to properly cut the preimage. Then the original signature does make more sense.